index.html

```diff
@@ -28,7 +28,136 @@ <h1>I’m Mariane Glad to see you!</h1>
         <li class="icons_list"><a href="#"><img src="./logos/Vector.png" alt="navigation icon"></a></li>
     </ul>
 </section>
+    <section class="cards">
+        <div class="card">
+            <img src="./images/SnapshootPortfolio.png" alt="navigation icon" class="snapshoot">
+             <h1>Tonic</h1>
+             
+             <div class="sec">
+                <p>CANOPY</p>
+                <span class="dot"></span>
+                <p class="sec_color1">Back End Dev</p>
+                <span class="dot"></span>
+                <p class="sec_color2">2015</p>
+            </div>
+            
+            <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="list">
+                <li class="box"><button type="button">html</button></li>
```

Comment:
- [ ] Please kindly make sure that the same font used in the original design is also used.
- [ ] Please ensure that the cards have a `border-radius` as it is in the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/164948468-cb13132c-46d9-4313-83de-57e41c053043.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/164948500-d154b6ab-cd32-4d47-b32e-57f87866ac56.png)